cli-plugins/apps/create-workload.hbs.md

```diff
@@ -127,9 +127,55 @@ tanzu apps workload create petclinic-image --param-yaml maven=$"artifactId:hello
 tanzu apps workload create petclinic-image --param-yaml maven="{"artifactId":"hello-world", "type": "jar", "version": "0.0.1", "groupId": "carto.run"}"
 ```
 
+## <a id='yaml-files'></a> Working with YAML files
+
+In many cases, you can manage workload life cycles through CLI commands.
+However, you might find cases where you want to manage a workload by using a `yaml` file.
+The Apps CLI plug-in supports using `yaml` files.
+
+The plug-in is designed to manage one workload at a time.
+When you manage a workload using a `yaml` file, that file must contain a single workload definition.
+Plug-in commands support only one file per command.
+
+For example, a valid file looks similar to the following example:
+
+```yaml
+---
+apiVersion: carto.run/v1alpha1
+kind: Workload
+metadata:
+  name: tanzu-java-web-app
+  labels:
+    app.kubernetes.io/part-of: tanzu-java-web-app
+    apps.tanzu.vmware.com/workload-type: web
+spec:
+  source:
+    git:
+      url: https://github.com/vmware-tanzu/application-accelerator-samples
+      ref:
+        tag: tap-1.3
+    subPath: tanzu-java-web-app
+```
+
+To create a workload from a file like the one just shown, run:
+
+```console
+tanzu apps workload create -f my-workload-file.yaml
+```
+
+Another way to create a workload from `yaml` is passing the definition through `stdin`. For example, run:
+
+```console
+tanzu apps workload create -f - --yes
```

Comment:
Is the --yes required in order for passing in via stdin to work properly?

If not, you didn't include `--yes` in the previous example which is good because we want the user to take their time going through this process and force them to notice and approve the diff.

If it would work without it, I'd recommend removing the `--yes` from this command example